eskipfile/remote.go

```diff
@@ -23,17 +23,24 @@ type remoteEskipFile struct {
 }
 
 type RemoteWatchOptions struct {
-	remoteFile    string
-	threshold     int
-	verbose       bool
-	failOnStartup bool
+	// URL of the route file
+	RemoteFile string
+
+	// Verbose mode for the dataClient
+	Verbose bool
+
+	// Amount of route changes that will trigger logs after route updates
+	Threshold int
```

Comment:
While we are on it, is this knob really necessary since it logs update sizes anyway?
https://github.com/zalando/skipper/blob/960ef409f56d38b13c2af2fb44f95751cc783d1e/eskipfile/remote.go#L110-L118

Also `if client.threshold >= 0` condition means it will warn-log even for default zero threshold which feels strange. I think the condition should be `> 0` but if we can get rid of unnecessary knob it would be the best. My concern is that once we make it public there is no way back.

Here https://github.com/zalando/skipper/blob/960ef409f56d38b13c2af2fb44f95751cc783d1e/eskipfile/remote.go#L91-L93
I think log level should be `Infof`

Reply:
I've modified the last to `Infof`, and also changed the condition to be `0 >' (with safer initialization).

The reason for the Warning to be there is because if suddenly there's a big jump on changed routes, we want to notice immediately (keep it mind this is downloading route files from a remote location, and you may even want to have certain monitoring connected to those warnings.

Reply:
> and you may even want to have certain monitoring connected to those warnings.

This is a good point, we may think (within separate scope) about exposing added/removed routes metric if that is interesting